### DISCUSSION TITLE
LINEから予約はできたのですが、Webから予約ができないのは仕様でしょうか?

### DISCUSSION THREAD

**Discussion by itsuki-yamada** [dzeyelid/onsen-maas-vueapp#20] [Q&A] [answered]
LINEから予約はできたのですが、Webから予約ができないのは仕様ですか?

ホストしているURL:https://lively-bay-00174bb00.azurestaticapps.net/

**Comment by mochan-tk** [comment]
webからでも予約はできるので、何かしらエラーがおきているようです。
（可能性としては、指定していただいたパラメータのうち、CosmosDBの値か、Messaging APIのチャネルIDとシークレットキーを再確認してみてくださいませ）

**Comment by itsuki-yamada** [comment, marked as answer]
ご回答ありがとうございます!
LINEの認証を通さずWEB画面が表示されていたようでした。
画面をリロードしたらLINEログイン画面がでてきて、予約できました!

**Comment by mochan-tk** [comment]
解決いただいたとのことで、良かったです！ご質問ありがとうございました！
